src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -172,6 +172,7 @@
 
     <PropertyGroup>
         <OtherRunnerScriptArgs Condition="'$(FilterToTargetGroup)' == 'net46'">$(OtherRunnerScriptArgs) --xunit-test-type=desktop </OtherRunnerScriptArgs>
+        <XunitArgs Condition="'$(FilterToTargetGroup)' == 'net46'"> -noshadow $(XunitArgs)</XunitArgs>
```

Comment:
@weshaggard yes I talked offline with Matt and he is already working to have unified targets for local and Helix runs. this change just to make sure we'll not have any breaks till Matt's finish his change.